test/functional/test-url-replacements.js

```diff
@@ -278,6 +278,16 @@ describes.sandboxed('UrlReplacements', {}, () => {
         });
   });
 
+  it('should replace CLIENT_ID with opt_cookieName', () => {
+    setCookie(window, 'url-abc', 'cid-for-abc');
+    // Make sure cookie does not exist
+    setCookie(window, 'url-xyz', '');
+    return expandAsync('?a=CLIENT_ID(abc,,url-abc)&b=CLIENT_ID(xyz,,url-xyz)',
```

Comment:
`cookieName` is only used when a cookie is written. `scope` is still used for all other thing, for e.g., `cidOnCache = sha384(baseCid + scope + domain)`